C/ds/Hash_Table.c

```diff
@@ -0,0 +1,133 @@
+    1                                                                                                                                               
+    2                                                                                                                                               
+    1                                                                                                                                               
+    3                                                                                                                                               
+    1                                                                                                                                               
+    12                                                                                                                                              
+    1                                                                                                                                               
+    4                                                                                                                                               
+    1                                                                                                                                               
+    8                                                                                                                                               
+    3  
+
+    Sample Output:
+    0 -> 0                                                                                                                                          
+    1 -> 0                                                                                                                                          
+    2 -> 2                                                                                                                                          
+    3 -> 3                                                                                                                                          
+    4 -> 12                                                                                                                                         
+    5 -> 4                                                                                                                                          
+    6 -> 0                                                                                                                                          
+    7 -> 0                                                                                                                                          
+    8 -> 8                                                                                                                                          
+    9 -> 0  
+
+*/
+
+/*
+    Time Complexity: O(SIZE)
+    Space Complexity: O(n) , n-no of values
+*/
```

Comment:
Add a new line here. Rest perfect

Reply:
A new line is already added. Should I add one more

Reply:
Yes, please.